int32 object_num
string object_name
---
bool success
